src/Alpha_complex/example/Weighted_alpha_complex_3d_from_points.cpp

```diff
@@ -34,10 +34,10 @@ int main(int argc, char **argv) {
     // ----------------------------------------------------------------------------
```

Comment:
I'm a total noob in C++, but is it normal that the comment says Complexity = FAST in line 10, and then we use complexity::SAFE on line 11? 
(Also for some reason I can only comment this code starting at line 34 ; cannot put my comment on the lines above...)

Reply:
There was an inconsistency between code and comment. Fixed on b1b0e9af